scripts/make-react.js

```diff
@@ -46,29 +29,23 @@ components.forEach((component) => {
   const eventImports = (component.events || [])
     .map(
       (event) =>
-        `import type { ${camelCaseToPascalCase(
-          event.eventName
-        )} } from '../../events/index.js';`
+        `import type { ${event.eventName} } from '../../events/index.js';`
     )
     .join("\n");
   const eventExports = (component.events || [])
     .map(
       (event) =>
-        `export type { ${camelCaseToPascalCase(
-          event.eventName
-        )} } from '../../events/index.js';`
+        `export type { ${event.eventName} } from '../../events/index.js';`
     )
     .join("\n");
   const eventNameImport =
     (component.events || []).length > 0
-      ? `import { EventName } from '@lit/react';`
+      ? `import type { EventName } from '@lit/react';`
       : ``;
   const events = (component.events || [])
     .map(
       (event) =>
-        `${event.reactName}: '${
-          event.name
-        }'  as EventName<${camelCaseToPascalCase(event.eventName)}>`
+        `${event.reactName}: '${event.name}'  as EventName<${event.eventName}>`
```

Comment:
I remember the events name being all messed up, thats why introduced that func, but as far as I can see in the react builds, its not anymore, what fixed it? Because earlier the name of the events were coming in camal case. Regardless its fixed now, will do more check by using the components soon :+1: